Leanplum-SDK/Classes/Leanplum.h

```diff
@@ -326,6 +327,15 @@ typedef enum {
         responseHandler:(LeanplumStartBlock)startResponse;
 /**@}*/
 
+/**
```

Comment:
Please let me know if this comment is acceptable. I'm happy to change it to whatever makes sense to your team.